projects/examples/package.json

```diff
@@ -3,14 +3,16 @@
   "main": "src/index.ts",
   "dependencies": {
     "@beanstalk/sdk": "workspace:*",
+    "@beanstalk/sdk-core": "workspace:*",
+    "@beanstalk/wells": "workspace:*",
```

Comment:
Adding this here but applies to `jest.config.ts` - should we add these new projects there? Along with top-level sdk-core and sdk-wells package.json commands (e.g. sdk-core:test).

Reply:
yeah, we can once we add tests